### PR TITLE
Add content to README

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

**Comment by robertkowalski** [comment]
@andywenk i addressed the issue and provided a walktrough. this depends on #34 - is there a reason not to merge it @KlausTrainer ?

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+```
+(sudo) couchdb
+
+```
+
+Given your username is `admin`, your password `secret` and the port of your Couch `5984` run:
+
+```
+COUCH=http://admin:secret@localhost:5984
+curl -X PUT $COUCH/_config/httpd_global_handlers/_authproxy \
```
Comment (KlausTrainer) [inline README.md]: I'd prefer to name the handler `_couch_email_auth`, as it's most obvious.

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+## Usage
+
+Send a `POST` request containing JSON to the endpoint:
+
+```
+curl -H Content-Type:application/json -X POST \
+http://localhost:3000 --data-binary '{"email": "andy@example.com","username": "Andy"}'
+```
+
+### Values for the request
```
Comment (KlausTrainer) [inline README.md]: I'd suggest "Request parameters" instead.

File: README.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+
+```
+email       (required)    The email address the token is getting mailed
+username    (optional)    The username of the user
+```
+
+## Templates
```
Comment (KlausTrainer) [inline README.md]: Assuming this shall be part of the "Usage" section, there is one `#` missing.

File: README.md [hunk 4]
```diff
@@ -1 +1,60 @@
...
+Change the configuration saved in `couch_email_authrc` to your needs, in this expample we are using port 3000 for the service.
+
+Start your couchdb:
+
+```
+(sudo) couchdb
+
+```
+
+Given your username is `admin`, your password `secret` and the port of your Couch `5984` run:
```
Comment (KlausTrainer) [inline README.md]: admin username

File: README.md (hunk 1 above)
Comment (robertkowalski) [inline README.md]: ok but we have to rename it when we add pluggable transports like SMS :)

**Comment by robertkowalski** [comment]
fixed everything!

**Comment by KlausTrainer** [comment]
Thanks!

**Pull request merged**